components/server/src/workspace/gitpod-server-impl.ts

```diff
@@ -2813,6 +2833,7 @@ export class GitpodServerImpl implements GitpodServerWithTracing, Disposable {
             { id: workspaceId },
             Permission.ADMIN_WORKSPACES,
         );
+        await this.auth.checkPermissionOnWorkspace(admin.id, "admin_control", workspaceId);
 
         const workspace = await this.workspaceDb.trace(ctx).findById(workspaceId);
```

Comment:
use `workspaceService.stopWorkspace`

Reply:
It does, internally. Once we rolled out FGA, "internalStopWorkspace" is replaced alltogether.